src/components/LoginButton.tsx

```diff
@@ -0,0 +1,24 @@
+import Button from "components/Button";
+import { backServer } from "loadenv";
+import { ButtonProps } from "components/Button";
+
+type LoginButtonProps = ButtonProps & { isLogin?: boolean; redirect: string };
+
+const LoginButton = (props: LoginButtonProps) => {
+  const { isLogin, redirect, ...buttonProps } = props;
+
+  return isLogin ? (
+    <Button {...buttonProps}>{props.children}</Button>
+  ) : (
+    <a
+      href={`${backServer}/auth/sparcssso?redirect=${encodeURIComponent(
+        redirect
+      )}`}
+      style={{ textDecoration: "none" }}
+    >
+      <Button {...buttonProps}>{props.children}</Button>
+    </a>
+  );
+};
+
+export default LoginButton;
```

Comment:
`Button`을 포함하는 컴포넌트보다는 `Link`와 비슷한 기능을 제공하는 컴포넌트를 만드는 거는 어떨까요??

```suggestion
const LinkLogin = () => {
  const { pathname, search } = useLocation();
  const redirectPath = pathname + search;
  return (
    <a
      href={`${backServer}/auth/sparcssso?redirect=${encodeURIComponent(
        redirectPath
      )}`}
      css={{ textDecoration: "none" }}
    >
      {children}
    </a>
  );
}
```

그리고 style object `css` prop으로 넘겨주세요 ~

Reply:
마찬가지로 `LinkLogout` 도 만들면 좋을 것 같아요

components/Link/index.tsx 에서
```
export { LinkLogin, LinkLogout }
```

Reply:
만들었습니다아 ~